README

```diff
@@ -127,6 +127,10 @@ interactive::
 Set to prompt a message and wait before testing the presence of a U2F device.
 Recommended if your device doesn't have a tactile trigger.
 
+[prompt=your prompt here without newline]::
+Set individual prompt message for interactive mode. Watch the square brackets
+around this parameter to get spaces correct recognized by PAM.
```

Comment:
an ok, native gerrman speaking so sometimes I miss the right syntax.

Yes, thats a pam thing: (last answer references the man page of it:
http://stackoverflow.com/questions/17731555/allow-escape-spaces-in-linux-pam-modules



> From the useful documentation available by typing `man pam.d`: (emphasis added)
> 
> module-arguments are a space separated list of tokens that can be used to modify the specific behavior of the given PAM. Such arguments will be documented for each individual module. **Note, if you wish to include spaces in an argument, you should surround that argument with square brackets.**